EqualSymbols/EqualSymbols.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
not necessary file

Reply:
Don't agree - solution file must be submitted